contents/docs/support-options.md

```diff
@@ -87,8 +93,6 @@ PostHog provides a free basic level of support to everyone via the community, wi
 </table>
 </div>
 
-By default, the Scale plan does not include items marked with a '+'.
-
 ## Community Support
 
 There are four ways to get free support:
```

Comment:
```suggestion
There are three ways to get community support:
```

Reply:
We should also delete the in-app option from this list, as it appears later and is confusing (as it's not community support, it's from PostHog)